dnd5esheets/front/src/store/index.ts

```diff
@@ -265,7 +265,7 @@ const effects = {
 };
 
 for (const derivedAttribute in effects) {
-  createEffect(() =>
+  createComputed(() =>
```

Comment:
For my own edification: what's the difference?

Reply:
Mainly:
> The first execution of the effect function is not immediate;
https://www.solidjs.com/docs/latest#createeffect

We might move on to `createMemo` later on, to avoid setting signals within the computed. I'll still getting familiar with all of these 😁